dmoj/cptbox/isolate.py

```diff
@@ -44,11 +44,19 @@ def __init__(self, read_fs, write_fs=None, writable=(1, 2)):
             self._getcwd_pid = lambda pid: os.readlink('/proc/%d/cwd' % pid)
             self._getfd_pid = lambda pid, fd: os.readlink('/proc/%d/fd/%d' % (pid, fd))
 
+        if has_landlock():
+            self.update({sys_openat: ALLOW, sys_open: ALLOW})
```

Comment:
Probably best to implement a debug mode that uses a callback to log all the `EACCES` generated by landlock. This will be highly useful for debugging executor failures.

Reply:
How would debug mode be set?

Reply:
Constructor argument, set to true during self-test.